definitions/firefox_desktop.toml

```diff
@@ -237,6 +237,45 @@ description = """
 """
 
 
+[metrics.daily_active_users]
+data_source = "clients_daily"
+select_expression = "COUNTIF(active_hours_sum > 0 AND total_uri_count > 0)"
```

Comment:
Do we need to add to note saying that if you DON'T aggregate by date, you are getting days of use ("or active user-days", or something?) Or does that just confuse things?I ask because this is really only "DAU" in one particular configuration - in all other cases you are opening up the possibility that a client counts more than once. 

I'm not blocking on this, but my 2c.

Reply:
Sounds good to me! I added "If the metric is NOT aggregated by `submission_date`, the metric is similar to a "days of use" metric."